Overlord.Api/Program.cs

```diff
@@ -20,7 +20,12 @@ public static void Main(string[] args)
 
             builder.Services.AddMqttBrokerServices();
 
-            var connectionString = builder.Configuration.GetConnectionString("OverlordContext");
+            string? connectionString = builder.Configuration.GetConnectionString("OverlordContext");
+
+            if (string.IsNullOrEmpty(connectionString))
+            {
+                throw new Exception("EMPTY CONNECTION STRING!");
+            }
```

Comment:
Lets only throw an exception with this specific message on envs that are not PROD